### ISSUE TITLE
Trying to get it to work

### ISSUE THREAD

**Issue by 3dfan** [alkaes/QidiPrint#3]
Maybe a first question, can this plugin be used to send print jobs also?

The plugin seems to load in Cura 4.7.1 according to the logs, however I do not have the plugin button (qidiprint)
Should I select a specific printer make and model? (I'm going with the custom XYZ at the moment)

cura 4.7.1 from ultimaker website on ubuntu 20.04.

[cura.log](https://github.com/alkaes/QidiPrint/files/5350752/cura.log)

**Comment by alkaes** [comment]
Sure it can be used to send print jobs, it was the first reason for making that plugin.
In order for the button to appear your printer definition json file must have a metadata manufacturer field set to "Qidi". 
Below is an example of printer definition json contents with correct manufacturer setting:
```
{
    "id": "X-MAX",
    "version": 2,
    "name": "X-MAX",
    "inherits": "ultimaker",
    "metadata": {
        "visible": true,
        "author": "Qidi",
        "manufacturer": "Qidi",
        "category": "Qidi",
        "file_formats": "text/x-gcode",
        "has_materials": true,
		"has_machine_quality": true,
        "preferred_quality_type": "normal",
        "icon": "icon_ultimaker2.png",
        "platform": "X-MAX.stl",
        "platform_offset": [-212, -36.3, 136],
        "_platform_texture": "Ultimaker2backplate.png",
        "first_start_actions": ["MachineSettingsAction"]
    },
    "overrides":{

		"machine_start_gcode":{ "default_value":"G92 A0 B0\nG28 X0 Y0\nG0 X-1000 Y-1000 F1800\nG28 Z0\nG1 X0 Y0 Z50 F3600\nG1 X0 Y0 F3600\nM140 S{material_bed_temperature_layer_0}\nM104 S{material_print_temperature_layer_0} T0\nM190 S{material_bed_temperature_layer_0}\nM109 S{material_print_temperature_layer_0} T0\nG92 E0\nG92 A0 B0\nG0 X0 Y4 Z0.3 F3600\nG1 X{machine_width} Y5 F2400\nG1 X5 A13 F2400\nG92 A0 B0\nG1 X5 F2400\n"},
		"machine_name": { "default_value": "X-MAX" },
        

        "machine_width": { "default_value": 300 },
        "machine_depth": { "default_value": 250 },
        "machine_height": { "default_value": 300 }

    }
}
```

**Comment by 3dfan** [comment]
Thank you for the quick reply.
I did start creating my own json for my X-Pro (dual nozzle) model, but I got stuck on some settings.
In the end I have installed the profiles from following git contribution and on first test they seem to be very complete and working with your plugin.
[qidi tech profiles](https://github.com/cjsatuforc/qidi-tech-cura)

![Screenshot from 2020-10-09 16-34-20](https://user-images.githubusercontent.com/16207861/95596596-df4c2100-0a4d-11eb-8fbf-ad32acd60ed0.png)
